File: sdks/php/src/Types/SubscriptionPlanResponseBillingInterval.php
<?php

namespace NovaBilling\Types;

enum SubscriptionPlanResponseBillingInterval: string
{
    case Hourly = "HOURLY";
    case Daily = "DAILY";
    case Weekly = "WEEKLY";
    case Monthly = "MONTHLY";
    case Quarterly = "QUARTERLY";
    case Yearly = "YEARLY";
}

File: sdks/swift/Sources/Schemas/SubscriptionPlanResponseBillingInterval.swift
import Foundation

public enum SubscriptionPlanResponseBillingInterval: String, Codable, Hashable, CaseIterable, Sendable {
    case hourly = "HOURLY"
    case daily = "DAILY"
    case weekly = "WEEKLY"
    case monthly = "MONTHLY"
    case quarterly = "QUARTERLY"
    case yearly = "YEARLY"
}